CHANGELOG.md

```diff
@@ -7,6 +7,17 @@
 
 # Release notes
 
+## 1.4.0
+
+### Technical Improvements
+
+* Fix app crashing due to missing empty constructor in OpenSettingsDialogFragment ([#366](https://github.com/medic/cht-android/pull/366))
+* Upgrade Ruby version from 2.7 to 3.3 for fastlane in build and publish files for GH actions ([#369](https://github.com/medic/cht-android/pull/369))
```

Comment:
It's a nicer experience to link to the issue rather than the PR...

```suggestion
* Fix app crashing due to missing empty constructor in OpenSettingsDialogFragment ([#338](https://github.com/medic/cht-android/issues/338))
* Upgrade Ruby version from 2.7 to 3.3 for fastlane in build and publish files for GH actions ([#368](https://github.com/medic/cht-android/issues/368))
```

Reply:
@garethbowen I've committed this one but the one in the `flavors` section has an issue in a private repository. Should I leave it as it is?

Reply:
Yeah that one will be fine as is.